seaclouds.yaml

```diff
@@ -28,8 +28,14 @@ services:
 - serviceType: brooklyn.entity.basic.SameServerEntity
   name: SeaClouds Dashboard + Planner
   brooklyn.children:
-  - serviceType: brooklyn.entity.webapp.tomcat.TomcatServer
+  - serviceType: brooklyn.entity.webapp.jetty.Jetty6Server
     name: SeaClouds Dashboard 
     id: dashboard
     war: https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=eu.seaclouds-project&a=dashboard&v=LATEST&e=war
     launch.latch: $brooklyn:component("deployer").attributeWhenReady("service.isUp")
+    brooklyn.config:
+      java.sysprops:
+        deployer.host: $brooklyn:component("deployer").attributeWhenReady("host.address")
+        deployer.httpPort: $brooklyn:component("deployer").attributeWhenReady("brooklynnode.webconsole.httpPort")
+        deployer.username: admin
```

Comment:
`$brooklyn:component("deployer").config("brooklynnode.managementUser")` and`$brooklyn:component("deployer").config("brooklynnode.managementPassword")` should help here ;)